### PR TITLE
Bump Django up to 2.1.15

### PR DESCRIPTION
**Pull request by Pomax**
This seems to fail a single test now, generating an HTTP200 success code, rather than HTTP400 error code, and that's rather puzzling.

### CODE REVIEW

**Comment by Pomax** [comment]
Haaaaa found it, and now I'm worried about how many posts we allowed that should not have >_>;;

File: pulseapi/entries/views.py [hunk 1]
```diff
@@ -525,10 +525,9 @@ def get_serializer_context(self):
     def post(self, request, *args, **kwargs):
         request_data = request.data
         user = request.user if hasattr(request, 'user') else None
-
         validation_result = post_validate(request)
 
-        if validation_result:
+        if validation_result is True:
```
Comment (Pomax) [inline pulseapi/entries/views.py]: @cadecairos so... yeah...  hmm.

Comment (cadecairos) [reply pulseapi/entries/views.py]: /facepalm

Comment (cadecairos) [reply pulseapi/entries/views.py]: "When JavaScript Developers start coding in Python"

Comment (Pomax) [reply pulseapi/entries/views.py]: Oh, no, this would **absolutely** be the same bug in JS, because `validation_result` can be `True` _or string data_

Comment (Pomax) [reply pulseapi/entries/views.py]: In fact, we had the correct check _in the same file, for the same var_ so I don't even know how this got here.

**Pull request merged**